src/lib/precomputed_block/precomputed_block.ml

```diff
@@ -0,0 +1,296 @@
+(* precomputed_block.ml *)
+
+open Core
+open Async
+
+(* Precomputed_block.t type has changed from mainnet, too hard to recreate old type
```

Comment:
Do you mean current `compatible` diverged from actual `mainnet`? Or the comment isn't up-to-date?

Reply:
It's not clear to me why this new file is in this PR